domain/src/main/java/com/bottlerocketstudios/mapsdemo/domain/models/UserFacingError.kt

```diff
@@ -0,0 +1,13 @@
+package com.bottlerocketstudios.mapsdemo.domain.models
+sealed class UserFacingError {
+    data class GeneralError(
+        val title: Int,
+        val description: Int
```

Comment:
@Josh-Gebbeken Annotate title and description with `@StringRes` to provide warnings if values that aren't string resources are passed in.